lib/identity.js

```diff
@@ -284,6 +284,20 @@ api.setRoles = function(actor, identity, callback) {
         actor, PERMISSIONS.IDENTITY_ADMIN, {resource: identity}, callback);
     },
     function(callback) {
+      // generate resource role resources
+      var roles = identity.sysResourceRole = identity.sysResourceRole || [];
```

Comment:
Not super sure why this was there in the first place (was used somehwere else), went ahead and removed it from both places.